crossplane-contrib/provider-tf-aws
# Remove region from iam group resource spec

### Description of your changes

IAM group resources do not have a region hence do not need a `region` field in their schema. This PR removes schema field from `iam` group resources.

Fixes #112 

I have:


### How has this code been tested

1. `kubectl create -f examples/iam/user.yaml`
2. Verify user created on the cloud side
3. `kubectl delete -f examples/iam/user.yaml`
4. Verify user deleted on the cloud side

[contribution process]: https://git.io/fj2m9
